Makefile

```diff
@@ -50,6 +50,11 @@ release:
 	GOOS=windows GOARCH=amd64 go build -ldflags "-X main.versionNumber=$(TRAVIS_TAG)" -o release/terraform-provider-alks_v$(TRAVIS_TAG).exe -mod=vendor $(package)
 	zip release/terraform-provider-alks-windows-amd64.zip release/terraform-provider-alks_v$(TRAVIS_TAG).exe
 
+	shasum -a 256 release/*.zip > release/terraform-provider-alks_v$(TRAVIS_TAG)_SHA256SUMS
```

Comment:
We produce both `tar` files and `zip` files depending on the platform (tar for *nix like, zip for windows).  Looks like right now this will only produce a signature for the windows zip files.

Reply:
Good catch! Added `release/*.tar.gz` and it now includes all the remaining files missing.